app/guid-node/registrations/controller.ts

```diff
@@ -48,13 +49,18 @@ export default class GuidNodeRegistrations extends Controller {
         this.set('schemas', schemas);
     });
 
-    @alias('model.taskInstance.value') node!: Node;
+    @alias('model.taskInstance.value') node!: Node | null;
```

Comment:
Yeah, this does feel weird, but, accounting to [the docs](http://ember-concurrency.com/docs/derived-state/) it will always be at least `null` and never `undefined` so I was trying to be explicitly correct.